script/lint

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
why this instead of pytest plugins?

Reply:
I was having issues with using them but I think that turned out to be a red herring. Also I thought it would be nice to have a script to run specifically for linting without having to include the full test suite

Reply:
I just checked and re-enabling the pytest plugins works fine locally, so if folks prefer we could go back to that

Reply:
i do prefer